src/material_library.h

```diff
@@ -0,0 +1,120 @@
+#include <stdlib.h>
+#include <iostream>
+#include <map>
+#include <string>
+#include <string>
+#include <vector>
+#include "material.h"
+
+namespace pyne {
+
+typedef std::map<std::string, pyne::Material> mat_map;
+typedef mat_map::iterator mat_iter;
+
+
+class MaterialLibrary {
+ protected:
+  // The actual library
+  mat_map material_library;  // material library
+  std::set<std::string> matlist;
```

Comment:
Good use of `std::set` here.

Reply:
As it's a set, why not name it as `matset` or `mat_set`?